src/vs/workbench/contrib/debug/browser/breakpointsView.ts

```diff
@@ -100,6 +104,10 @@ export class BreakpointsView extends ViewPane {
 		this._register(this.menu);
 		this.breakpointItemType = CONTEXT_BREAKPOINT_ITEM_TYPE.bindTo(contextKeyService);
 		this.breakpointSupportsCondition = CONTEXT_BREAKPOINT_SUPPORTS_CONDITION.bindTo(contextKeyService);
+		this.breakpointSupportsReadAccess = CONTEXT_BREAK_WHEN_VALUE_IS_READ_SUPPORTED.bindTo(contextKeyService);
```

Comment:
I do not understand why are we setting these context keys in the breakpoints view, since we are only using them in the `variablesView` for command enablment. Can all of this work in the breakpoints view be removed, or am I missing some connection?

Reply:
You are correct, it was previously used to update breakpoints in the `Breakpoints` view, as checkboxes. Since I removed previous code about it, I must remove this part as well.